### PR TITLE
Document `app.setName()`

### PR DESCRIPTION
**Pull request by k0kubun**
I guess it was implemented in https://github.com/atom/electron/commit/5670ee7693dc4f3cb6bb95131752852693b71eec.

### CODE REVIEW

File: docs/api/app.md [hunk 1]
```diff
@@ -331,6 +331,10 @@ to the npm modules spec. You should usually also specify a `productName`
 field, which is your application's full capitalized name, and which will be
 preferred over `name` by Electron.
 
+### `app.setName()`
```
Comment (kevinsawicki) [inline docs/api/app.md]: I think the `name` parameter to this function should be documented and mention that it should be a `String`

The docs for `app.setPath` have an example of the format, https://github.com/k0kubun/electron/blob/app-doc-set-name/docs/api/app.md#appsetpathname-path

Comment (k0kubun) [inline docs/api/app.md]: Nice catch. Unified the format and squashed in https://github.com/atom/electron/pull/4851/commits/8cf4574198da33976ffc0e3c6d87d6aac6e6f48c

**Comment by kevinsawicki** [comment]
Thanks for adding this :memo: :+1:

**Comment by k0kubun** [comment]
Thank you for merging too! I'm always helped by this doc.

**Pull request merged**